backend/src/v5/utils/helper/yup.js

```diff
@@ -100,12 +100,13 @@ YupHelper.types.strings.name = Yup.string().min(1).max(35);
 
 YupHelper.types.date = Yup.date().transform((n, orgVal) => new Date(orgVal));
 
-YupHelper.types.embeddedImage = Yup.mixed().transform((n, orgVal) => (orgVal ? Buffer.from(orgVal, 'base64') : n))
-	.test('Image', `must be smaller than ${fileUploads.resourceSizeLimit} Bytes`, (buffer) => !buffer || buffer.length <= fileUploads.resourceSizeLimit)
-	.test('Image', `must be of type ${fileUploads.imageExtensions.join(',')}`, async (buffer) => {
+YupHelper.types.embeddedImage = (isNullable) => Yup.mixed().transform((n, orgVal) => (orgVal ? Buffer.from(orgVal, 'base64') : n))
+	.test('Too large image', `Image must be smaller than ${fileUploads.resourceSizeLimit} Bytes`, (buffer) => !buffer || buffer.length <= fileUploads.resourceSizeLimit)
+	.test('Non supported type', `Image must be of type ${fileUploads.imageExtensions.join(',')}`, async (buffer) => {
 		if (!buffer) return true;
 		const ext = await fileExtensionFromBuffer(buffer);
 		return ext && fileUploads.imageExtensions.includes(ext.toLowerCase());
-	});
+	})
+	.test('Not null', 'Image cannot be null', (val) => isNullable || val !== null);
```

Comment:
So because mixed() allows null I tried to change embeddedImage to object but Yup doesnt like it

![image](https://user-images.githubusercontent.com/71635617/203072293-6db07e05-eaf9-4e0d-9bd2-0cc89b165aab.png)